src/components/DatePicker/DatePicker.st.css

```diff
@@ -0,0 +1,581 @@
+}
+
+
+/******************************************************/
+
+
+/* month and years dropdowns */
+
+.calendar::header::monthDropdown::caption ,
+.calendar::header::yearDropdown::caption {
+    font: value(CurrentMonthAndYearCaptionFont);
+    font-size: 16px;
+    color: value(CurrentMonthAndYearCaptionColor);
+    height: 24px;
+}
+
+/* List */
+
+.calendar::header::monthDropdown::dropdown::list::contentContainer,
+.calendar::header::yearDropdown::dropdown::list::contentContainer {
+    background-color: value(CurrentBackgroundColor);
+    padding: 12px 0;
+}
+
+/* List: Scroll */
+.calendar::header::monthDropdown::dropdown::list::options::-webkit-scrollbar,
+.calendar::header::yearDropdown::dropdown::list::options::-webkit-scrollbar {
+    width: 16px;
+}
+
+.calendar::header::monthDropdown::dropdown::list::options::-webkit-scrollbar-thumb,
+.calendar::header::yearDropdown::dropdown::list::options::-webkit-scrollbar-thumb {
+    /* https://stackoverflow.com/a/16829330/5343330 */
+    background-color: value(DropdownOptionSelectedAndHoverBGColor);
+    background-clip: content-box;
+    border-radius: calc(4px * 2);
+    border: 4px solid transparent;
+    height: 72px;
+}
+
+.calendar::header::monthDropdown::dropdown::list::options::-webkit-scrollbar-thumb:hover,
+.calendar::header::yearDropdown::dropdown::list::options::-webkit-scrollbar-thumb:hover {
+    background-color: applyOpacity(value(CurrentDayColor), 0.15);
+}
+
+.calendar::header::monthDropdown::dropdown::list::options::-webkit-scrollbar-thumb:active,
+.calendar::header::yearDropdown::dropdown::list::options::-webkit-scrollbar-thumb:active {
+    background-color: applyOpacity(value(CurrentDayColor), 0.25);
+}
+/*************************/
+
+.calendar::header::monthDropdown::dropdown::list::selectableOption,
+.calendar::header::yearDropdown::dropdown::list::selectableOption {
+    background-color: value(CurrentBackgroundColor);
+    padding: 0 20px;
+}
+
+.calendar::header::monthDropdown::dropdown::list::selectableOption:not(:disabled):not(:selected):hover,
+.calendar::header::yearDropdown::dropdown::list::selectableOption:not(:disabled):not(:selected):hover {
+    background-color: value(DropdownOptionSelectedAndHoverBGColor);
+}
+
+.calendar::header::monthDropdown::dropdown::list::selectableOption:selected,
+.calendar::header::yearDropdown::dropdown::list::selectableOption:selected {
+    background-color: value(DropdownOptionSelectedAndHoverBGColor);
+}
+
+.calendar::header::monthDropdown::dropdown::list::selectableOption:selected:hover,
+.calendar::header::yearDropdown::dropdown::list::selectableOption:selected:hover {
+    background-color: value(DropdownOptionSelectedAndHoverBGColor);
+}
+
+.calendar::header::monthDropdown::dropdown::list::selectableOption::title,
+.calendar::header::yearDropdown::dropdown::list::selectableOption::title {
+    color: value(CurrentDayColor);
+    font: font("{theme: 'Body-M', size: '14px', lineHeight: '20px'}");
```

Comment:
Not critical at all, but is there a reason why the font is hard-coded here?
Is this cause it's used only here?
I wish we would have IDE support, so that we could really use consts effectively

Reply:
Yes, it only used once, and there is no option to override it.